shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/resources/ManagedProcessor.java

```diff
@@ -110,20 +122,15 @@ public ManagedProcessor build() {
                             .buildWithDefaults(LabelsBuilder.V2_OPERATOR_NAME))
                     .build();
 
-            ManagedProcessorSpec ManagedProcessorSpec = new ManagedProcessorSpec();
-            ManagedProcessorSpec.setId(processorId);
-            ManagedProcessorSpec.setBridgeId(bridgeId);
-            ManagedProcessorSpec.setShardId(customerId);
-            ManagedProcessorSpec.setName(processorName);
+            ManagedProcessorSpec managedProcessorSpec = new ManagedProcessorSpec();
+            managedProcessorSpec.setId(processorId);
+            managedProcessorSpec.setBridgeId(bridgeId);
+            managedProcessorSpec.setName(processorName);
 
```

Comment:
@lucamolteni Can you please add some `Labels` to the the `metadata` of the `ManagedProcessor` that lets us quickly find them using `kubectl`. I think the following labels would be useful

- smartevents.services.redhat.com/customer-id
- smartevents.services.redhat.com/bridge-id
- smartevents.services.redhat.com/processor-id

And maybe:

- smartevents.services.redhat.com/processor-name

This will support things like `kubectl get ManagedProcessor -lsmartevents.services.redhat.com/customer-id=rob,smartevents.services.redhat.com/processor-id=my-processor-id`

Some of these labels already exist in the v2 `LabelsBuilder`, but you'll need to add the `Processor` specific ones.